src/reducers/fileTree.spec.tsx

```diff
@@ -20,11 +21,15 @@ describe(__filename, () => {
     it('builds and loads a tree', () => {
       const version = createInternalVersion(fakeVersion);
       const state = reducer(undefined, actions.buildTree({ version }));
+      const nodes = buildFileTree(version);
 
       expect(state).toEqual({
         ...initialState,
         forVersionId: version.id,
-        tree: buildFileTree(version),
+        tree: {
```

Comment:
I'm seeing this pattern of calling two functions repeatedly. Why not make it one?

```tsx
      expect(state).toEqual({
        ...initialState,
        forVersionId: version.id,
        tree: buildFileTree(version),
      });
```

This would mean renaming the current `buildFileTree()` to `buildFileTreeNode()` and adding a new function, `buildFileTree()`, that combines everything and returns the final tree.